src/components/ui/loading.tsx

```diff
@@ -0,0 +1,22 @@
+import { cn } from "@/lib/utils";
```

Comment:
Is this component part of shadcn/ui? If not it should live outside this `ui` directory I think. This is generally used for components added by https://ui.shadcn.com/docs/cli

Reply:
gotcha, thank you for the clarification. This has been moved